app/ui-react/syndesis/src/modules/integrations/components/editor/makeEditorResolvers.ts

```diff
@@ -189,13 +189,15 @@ export function makeEditorResolvers(esr: typeof stepRoutes) {
       >(esr.connection.describeData, configureDescribeDataShapeMapper),
     },
     apiProvider: {
-      upload: makeResolver<
+      editSpecification: makeResolver<
         IEditorConfigureStep,
         IConfigureStepRouteParams,
         IConfigureStepRouteState
-        >(esr.apiProvider.upload, configureConfigureStepMapper),
-      review: makeResolverNoParams('todo review'),
-      edit: makeResolverNoParams('todo edit'),
+      >(esr.apiProvider.editSpecification, configureConfigureStepMapper),
+      selectMethod: makeResolverNoParams('todo select method'),
```

Comment:
It's like a journey, I think the next stop is here :joy_cat:

Reply:
Right? This is the first page so it's definitely the next stop lol